templates/templates/blockified/single-product.html

```diff
@@ -52,6 +52,8 @@ <h2 class="wp-block-heading">Related products</h2>
 
 			<!-- wp:post-title {"textAlign":"center","level":3,"fontSize":"medium","__woocommerceNamespace":"woocommerce/product-query/product-title"} /-->
 
+			<!-- wp:woocommerce/product-rating {"isDescendentOfSingleProductTemplate":true} /-->
```

Comment:
You added the product rating to the Related Products Block. Should we add the block at line 16?

Reply:
Ohh I'm sorry, you're correct, it is in the wrong position. I just pushed a new commit fixing that. Thank you for pointing this out